gamemode/modules/base/sv_data.lua

```diff
@@ -236,6 +236,16 @@ function DarkRP.retrieveRPNames(name, callback)
     end)
 end
 
+function DarkRP.offlinePlayerData(steamid, callback, failed)
+    local split = string.Explode(":",steamid:gsub("STEAM_",""))
+    if (not split[1] or not split[2] or not split[3]) then failed() return end
+    local uniqueid = util.CRC("gm_STEAM_" .. split[1] .. ":" .. split[2] .. ":" .. split[3] .. "_gm")
```

Comment:
What error will be thrown when the SteamID is not of the format `STEAM_nr_nr:nr`?